### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by jasonday** [marcantondahmen/automad#75]
Current known issues (not exhaustive, I will add to over the next few days along with potential fixes)

* No visual focus indicator for interactive elements
* Images don't have an option for alt text
* Theme hamburger menu <a> doesn't convey state. This should be a button and indicate `aria-expanded=[true/false]`
* Next/previous links (from pages to other pages) - have poor color contrast on focus/active state
*

**Comment by jasonday** [comment]
@marcantondahmen any update here?
